doc/misc.rst

```diff
@@ -81,12 +84,6 @@ WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 OTHER DEALINGS IN THE SOFTWARE.
 
-Frequently Asked Questions
-==========================
-
-The FAQ is maintained collaboratively on the
-`Wiki FAQ page <http://wiki.tiker.net/BoxTree/FrequentlyAskedQuestions>`_.
-
```

Comment:
The link here is dead, so this just removed it. Is there a newer location for some FAQ?

Reply:
Not really. Could be in the docs.